### ISSUE TITLE
Install error on macOS 10.12.2 beta

### ISSUE THREAD

**Issue by romanserazhiev** [docker/for-mac#975]
### Expected:
Docker is installed

### Actual:
Docker displays an error

### Information

Docker for Mac: version: 1.12.3 (583d1b8)
OS X: version 10.12.2 (build: 16C48b)
logs: /tmp/E52E7F57-3C26-4551-A181-AF08B4966C00/20161122-120206.tar.gz
failure: com.docker.slirp is not running
[ERROR]  vmnetd
         /var/tmp/com.docker.vmnetd.socket does not exist
         Unexpected error (No such file or directory) connecting to /var/tmp/com.docker.vmnetd.socket
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux -db is not running
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         Connection refused (ECONNREFUSED) connecting to /Users/roman/Library/Containers/com.docker.docker/Data/s40: check if service is running
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Connection refused (ECONNREFUSED) connecting to /Users/roman/Library/Containers/com.docker.docker/Data/s51: check if service is running
         com.docker.slirp is not running
         slirp check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         cannot find docker
         docker-cli check failed with: (Failure "docker -v: timeout after 10.00s")
[OK]     menubar
[OK]     disk
Failure: Could not upload diagnostic data to remote server (docker-diagnose exit code is 1)

### Steps to reproduce the behavior

  1. Download Docker for Mac (stable or beta)
  2. Drag the application file to Applications folder
  3. Launch Docker
  4. Enter the password

**Comment by romanserazhiev** [comment]
Just wanted to add a comment to draw the whole picture. I used to have the beta of Docker app installed and working correctly without issues. Originally Docker (Toolbox?) was installed with brew (I think).

I decided to uninstall Docker from Brew and system completely, and re-install the Docker app. After doing so, this issue appeared. When I try docker -version, Terminal says that that docker is unknown command. I assume that when I give my admin password to Docker app, it should install it, right?
